docs/developer-docs/smart-contracts/advanced-features/management-canister.mdx

```diff
@@ -0,0 +1,123 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ICP Management Canister
+
+## Overview
+
+The ICP management canister is used to provide access to system features, such as managing canisters, integrating with bitcoin, making HTTP outcalls, signing with tECDSA, and securely retrieving randomness.
+
+
+The management canister is a virtual canister in the sense that it does not exist with an isolated state or Wasm module, but rather is implemented as part of ICP itself. For most practical purposes, the management canister behaves like a canister, i.e. it can accept messages from users or other canisters and can respond to those messages just like any other canister.
+
+
+The address of the ICP management canister is `aaaaa-aa` (i.e. the empty blob). It is present on any ICP subnet. By using this address, a canister can  access the management canister either on the local or remote subnet -- ICP will take care of routing the request to the appropriate subnet transparently. The cost of making the call is charged to the calling canister just like any other inter-canister call.
```

Comment:
> We might want to add a sentence that certain functionality like threshold signing and access to the Bitcoin network is only implemented on certain subnets.

Yeah sure.

> A graphic could also illustrate that the management canister abstracts functionalities of other canisters behind the scenes.

This is only the case for the bitcoin apis and I think it might be too advanced, even for an advanced features page and could even create extra confusion.

Reply:
I added a note about 

> We might want to add a sentence that certain functionality like threshold signing and access to the Bitcoin network is only implemented on certain subnets.